astropy/io/ascii/tests/test_html.py

```diff
@@ -16,7 +16,7 @@
 
 from .common import setup_function, teardown_function
 from ....tests.helper import pytest
-from ....extern.six.moves import cStringIO
+from ....extern.six.moves import range, cStringIO as StringIO
```

Comment:
Because it's now called `StringIO`. Like `xrange`, `izip` and `imap` it's a name that isn't used anymore in python3.